lib/tests/distributions/WDDistributionTest.m

```diff
@@ -108,7 +108,7 @@ function testWDDistribution(testCase)
             testCase.verifyEqual(wdRew.w, wNew/sum(wNew));
             
             %% convolution
-            wd2 = WDDistribution([0, 4, 5, 6], [1, 3, 4, 2]);
+            wd2 = WDDistribution([0, 4, 5, 6], [1, 3, 4, 2]/10);

```

Comment:
It is not necessary but a warning is thrown otherwise. Since I have set Jenkins to strict mode, the test case counts as failed if a warning comes up. This was the quickest fix, though it would also be possible to suppress the warning for this test case.